### PR TITLE
Move blocked_file_types configuration

### PR DESCRIPTION
**Pull request by luca-rath**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | yes
| Fixed tickets | -
| Related issues/PRs | -
| License | MIT
| Documentation PR | https://github.com/sulu/sulu-docs/pull/623

#### What's in this PR?

Move configuration `sulu_media.format_manager.blocked_file_types` to `sulu_media.upload.blocked_file_types`

### CODE REVIEW

File: src/Sulu/Bundle/MediaBundle/DependencyInjection/SuluMediaExtension.php [hunk 1]
```diff
@@ -412,7 +412,11 @@ private function configureFileValidator(array $config, ContainerBuilder $contain
         $definition = $container->getDefinition('sulu_media.file_validator');
         $definition->addMethodCall('setMaxFileSize', [$config['upload']['max_filesize'] . 'MB']);
 
-        $blockedFileTypes = $config['format_manager']['blocked_file_types'];
+        $blockedFileTypes = $config['upload']['blocked_file_types'];
+        if (0 === \count($blockedFileTypes)) {
+            $blockedFileTypes = $config['format_manager']['blocked_file_types'];
```
Comment (alexander-schranz) [inline src/Sulu/Bundle/MediaBundle/DependencyInjection/SuluMediaExtension.php]: there is also a parameter set with `sulu_media.media.blocked_file_types` which should also contain the new value. Not sure why not this parameter was use in the service definition 🙈.

**Review by alexander-schranz — CHANGES_REQUESTED** [review]
We shoudl also create a test in the `SuluMediaExtensionTest` to be sure that the implemented behaviour works.

**Review by alexander-schranz — APPROVED** [review]

**Comment by alexander-schranz** [comment]
Would be nice if we add a `Configuration` headline docs in the [MediaBundle Documentation](https://docs.sulu.io/en/2.2/bundles/media/index.html).

**Comment by niklasnatter** [comment]
Looks good. Would be nice to add a `Configuration` section to the docs (See: this:https://docs.sulu.io/en/2.2/bundles/preview/index.html#configuration)

**Comment by luca-rath** [comment]
Already on it 🙂

**Pull request merged**